src/types/Todo.ts

```diff
@@ -0,0 +1,9 @@
+import { User } from './User';
+
+export interface Todo {
+  id: number,
```

Comment:
Consider renaming `id` to `todoId` for better clarity and consistency with other models. 🤔